examples/src/main/java/io/grpc/examples/experimental/CompressionServerInterceptor.java

```diff
@@ -0,0 +1,18 @@
+package io.grpc.examples.experimental;
```

Comment:
This needs the copyright header.

Reply:
Moved into CompressingHelloWorldServerAllMethods as the review comment below.